paasta_tools/tron_tools.py

```diff
@@ -670,10 +668,7 @@ def validate(self):
         error_msgs.extend(super().validate())
         # Tron is a little special, because it can *not* have a deploy group
         # But only if an action is running via ssh and not via paasta
-        if (
-            self.get_deploy_group() is None
-            and self.get_executor() in MESOS_EXECUTOR_NAMES
-        ):
+        if self.get_deploy_group() is None and self.get_executor() != "ssh":
```

Comment:
I tweaked this manually. This will make this check stricter -- anything with executor: spark will now have this check.